components/brave_wallet_ui/common/hooks/use-balances-fetcher.tsx

```diff
@@ -36,10 +36,7 @@ export const useBalancesFetcher = (arg: Arg | typeof skipToken) => {
       arg.networks.length
       ? {
           accountIds: arg.accounts.map((account) => account.accountId),
-          networks: arg.networks.map(({ chainId, coin }) => ({
-            chainId,
-            coin
-          })),
+          networks: arg.networks,
```

Comment:
We should not pass in more data than is needed for query params. as all argument data is serialized and stored.

Reply:
https://github.com/brave/brave-core/pull/20812/commits/419eda221b83c47e8041e3eb0ef9d94a46c772e5